### ISSUE TITLE
Add a CI

### ISSUE THREAD

**Issue by FagnerMartinsBrack** [impress/impress.js#560]
We need to add a continuous integration service that runs the test whenever there is a new Pull Request. Only one, just to start, probably [Travis CI](https://travis-ci.com) that is free for Open Source.

**Comment by FagnerMartinsBrack** [comment]
In order to make Travis CI working we need to make the tests run on Linux, currently it seems the [`chrome` package](npmjs.com/package/chrome) we are using to run chrome locally in the command line [only works in Mac](https://github.com/afc163/chrome/blob/e3720fc584c95eb79dea2e5b4c5cd9c6726176a1/index.js#L9).

I have tried running the QUnit tests using Karma locally, but the complexity and amount of code seems too high. Therefore, I am looking into a simpler solution that doesn't require the project to figure out how to make the tests work locally in Windows, Mac and Linux Ubuntu (for Travis CI).

I have tried drone.io as @bartaz mentioned [in his comment](https://github.com/impress/impress.js/issues/435#issuecomment-177916641), but after I created the account I noticed they use outdated node versions `0.8` and `0.10` only. We can't rely in a service that doesn't update the node version because there are lots of packages we might depend upon that doesn't work in those versions anymore and we can stop receiving important updates and features that will stop the progress.

I have tried using [gulp](http://npmjs.com/package/gulp), so that we can find some plugin that can run the tests out of the box inside a service (like Sauce Labs*). I have found [`gulp-qunit`](https://www.npmjs.com/package/gulp-qunit), but unfortunately it doesn't work because it runs in PhantomJS and [Phantom JS don't support CSS 3D transforms](https://github.com/ariya/phantomjs/issues/12779), not even PhantomJS 2. When running `gulp-qunit`, the [QUnit integration tests](https://github.com/impress/impress.js/tree/a1d6475bd5f31b46586ae73a5d6e995d3ca4732b/test) fail locally:

```shell
➜  impress.js git:(ci) ✗ ./node_modules/.bin/gulp test-integration
[16:47:16] Using gulpfile ~/Git/impress.js/gulpfile.js
[16:47:16] Starting 'test-integration'...
[16:47:19] Testing core_tests.html
[16:47:19] Test failed: Initialization: Attributes
[16:47:19] Failed assertion: canvas.style.transform initialized correctly, expected: rotateZ(0deg) rotateY(0deg) rotateX(0deg) translate3d(1000px, 0px, 0px), but was: undefined
[16:47:19] at file:////Users/fagnerbrack/Git/impress.js/test/core_tests.js:50:25
[16:47:19] Failed assertion: canvas.style.transformOrigin initialized correctly, expected: left top 0px, but was: undefined
[16:47:19] at file:////Users/fagnerbrack/Git/impress.js/test/core_tests.js:54:25
[16:47:19] Failed assertion: canvas.style.transformStyle initialized correctly, expected: preserve-3d, but was: undefined
[16:47:19] at file:////Users/fagnerbrack/Git/impress.js/test/core_tests.js:58:25
[16:47:19] 
[16:47:19] Took 3071ms to run 22 tests. 19 passed, 3 failed.
[16:47:19] gulp-qunit: ✖ QUnit assertions failed in core_tests.html
[16:47:19] 'test-integration' errored after 3.47 s
[16:47:19] Error in plugin 'gulp-qunit'
Message:
    Command failed: /Users/fagnerbrack/Git/impress.js/node_modules/phantomjs2/lib/phantom/bin/phantomjs /Users/fagnerbrack/Git/impress.js/node_modules/qunit-phantomjs-runner/runner.js file:////Users/fagnerbrack/Git/impress.js/test/core_tests.html

Details:
    killed: false
    code: 1
    signal: null
    cmd: /Users/fagnerbrack/Git/impress.js/node_modules/phantomjs2/lib/phantom/bin/phantomjs /Users/fagnerbrack/Git/impress.js/node_modules/qunit-phantomjs-runner/runner.js file:////Users/fagnerbrack/Git/impress.js/test/core_tests.html
```

Any useful ideas on how to make tests working in Linux Ubuntu (for Travis CI) with the least amount of effort will be appreciated.

<sup>* The Sauce approach is how it is already being done with [js-cookie](https://saucelabs.com/u/js-cookie), although it uses grunt instead of gulp and [Sauce Labs is known to become unstable sometimes](https://github.com/js-cookie/js-cookie/pull/179) (even bootstrap is looking for changing it, see https://github.com/twbs/bootstrap/issues/18754).</sup>

**Comment by FagnerMartinsBrack** [comment]
After #566 this might become easier. I have received some tips to use CircleCI instead of Travis, seems to be better at running Chrome. Need to take a look on that.

**Comment by nkbt** [comment]
Added some PR's related to this, @FagnerMartinsBrack

**Comment by FagnerMartinsBrack** [comment]
Fixed by https://github.com/impress/impress.js/pull/568.
